src/models/data.py

```diff
@@ -694,14 +721,17 @@ def ds_folder_to_np(
 
         x, y = self.apply_spatial_mask(x, y)
 
-        yearly_agg = self._calculate_aggs(
-            x
-        )  # before to avoid aggs from surrounding pixels
+        if self.incl_yearly_aggs:
+            assert False
```

Comment:
just put this in because i wanted to check that the flag was working - REMOVE BEFORE END

Reply:
Not removed